src/main/java/seedu/address/ui/EventDetailView.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.ui;
+
+import java.time.format.DateTimeFormatter;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import seedu.address.model.event.Event;
+
+/**
+ * A UI component that displays detailed information about an {@code Event}.
+ * This panel updates dynamically when a new {@code Event} is provided.
+ */
+public class EventDetailView extends UiPart<Region> {
+    private static final String FXML = "EventDetailView.fxml";
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("MMM d, yyyy");
+
+    @FXML
+    private Label title;
+
+    @FXML
+    private Label date;
+
+    @FXML
+    private VBox attendees;
+
+    public EventDetailView() {
+        super(FXML);
+    }
+
+    /**
+     * Sets the details of the given {@code Event} to be displayed in this panel.
+     *
+     * @param event The event whose details are to be displayed.
+     */
+    public void setEvent(Event event) {
```

Comment:
Perhaps we could standardise the class to implement `DetailView<Event>` and override the `update` method.

Reply:
Makes sense, I'll change it